anchore_engine/clients/syft_wrapper.py

```diff
@@ -23,45 +17,6 @@ def run_syft(image):
 
     cmd = "syft -vv -o json oci-dir:{image}".format(image=image)
 
-    logger.debug("running syft: cmd={}".format(repr(cmd)))
-    rc, stdout, stderr = run_command_list(shlex.split(cmd), env=proc_env)
-    logger.debug(
-        "results: rc={} stdout={} stderr={}".format(rc, repr(stdout), repr(stderr))
-    )
-
-    if rc != 0:
-        raise SyftError(cmd=cmd, rc=rc, out=stdout, err=stderr)
+    stdout, _ = run_check(shlex.split(cmd), env=proc_env)
```

Comment:
related to https://github.com/anchore/anchore-engine/pull/740/files#r532760445 , if not logged there, this would be the next best spot to log syft output, but there currently is none.